docs/topics/cli-args.adoc

```diff
@@ -184,4 +184,7 @@ A space-delimited list of the packages to be evaluated by {ProductShortName}. It
 * In most cases, you are interested only in evaluating custom application class packages and not standard Java EE or third party packages. The `<PACKAGE_N>` argument is a package prefix; all subpackages will be scanned. For example, to scan the packages `com.mycustomapp` and `com.myotherapp`, use `--packages com.mycustomapp com.myotherapp` argument on the command line.
 * While you can provide package names for standard Java EE third party software like `org.apache`, it is usually best not to include them as they should not impact the migration effort.
 
-WARNING: If you omit the `--packages` argument, every package in the application is scanned, which can impact performance. It is best to provide this argument with one or more packages.
+[WARNING]
+====
+If you omit the `--packages` argument, every package in the application is scanned, which can impact performance. It is best to provide this argument with one or more packages.
```

Comment:
I think you can remove the second sentence of the warning. fluff. It's obvious from the first sentence that the user should use it. And they can't use the argument UNLESS they use it with one or more packages. 

You could change it to "You can specify multiple packages." That gets across the point that the user can specify more than one, without the "It is best to ...." Also, that kind of wording is not recommended. Use prescriptive wording rather than phrasing as a recommendation.